compiler/damlc/daml-stdlib-src/DA/BigNumeric.daml

```diff
@@ -0,0 +1,54 @@
+-- Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+{-# LANGUAGE CPP #-}
+
+-- | HIDE
+module DA.BigNumeric where
+
+#ifdef DAML_BIGNUMERIC
+-- TODO https://github.com/digital-asset/daml/issues/8719
+--  clean this module
+
+import GHC.Types (primitive)
+
+roundingUp, roundingDown, roundingCeiling, roundingFloor, roundingHalfUp, roundingHalfDown, roundingHalfEven, roundingUnnecessary: RoundingMode
+roundingUp = primitive @"BERoundingUp"
+roundingDown = primitive @"BERoundingDown"
+roundingCeiling = primitive @"BERoundingCeiling"
+roundingFloor = primitive @"BERoundingFloor"
+roundingHalfUp = primitive @"BERoundingHalfUp"
+roundingHalfDown = primitive @"BERoundingHalfDown"
+roundingHalfEven = primitive @"BERoundingHalfEven"
+roundingUnnecessary = primitive @"BERoundingUnnecessary"
+
+zero, one, ten: BigNumeric
+zero = fromNumeric (0.0: Numeric 0)
+one = fromNumeric (1.0: Numeric 0)
+ten = fromNumeric (10.0: Numeric 0)
```

Comment:
What is this for?

Reply:
It is just useful constants. For instance,  java.math.BigDecimal defines ZERO, ONE, TEN.
Whoever will clean this file is free to dropped them.